README.md

```diff
@@ -2,7 +2,7 @@ Nomad [![Build Status](https://circleci.com/gh/hashicorp/nomad.svg?style=svg)](h
 =========
 
 <p align="center" style="text-align:center;">
-  <img src="https://cdn.rawgit.com/hashicorp/nomad/master/website/source/assets/images/logo-text.svg" width="500" />
+  <img src="https://github.com/hashicorp/nomad/blob/19c404ca791d6ebe95a81738d7dc6623ab28564d/website/public/img/logo-text.svg" width="500" />
```

Comment:
Huh, this works as an img src even though navigating directly to the page isn't just an image?

Are https://raw.githubusercontent.com URLs usable? https://raw.githubusercontent.com/hashicorp/nomad/19c404ca791d6ebe95a81738d7dc6623ab28564d/website/public/img/logo-text.svg

Reply:
can we just inline the SVG?

Reply:
Strange, but it does seem to work: https://github.com/hashicorp/nomad/tree/fix-readme-img

When you request from the home page the `Content-Type` header is set properly, but if you access directly it's not ¯\_(ツ)_/¯

Reply:
Maybe leverage the website and point to https://nomadproject.io/img/logo-hashicorp.svg?

Reply:
That relies on us never changing the image location, which seems brittle maybe?

vault uses this approach, which I think works for now, although it may be a bit slower. We can add an issue to come up with a more permanent, fast solution
```
 <img width="300" alt="Vault Logo" src="https://github.com/hashicorp/vault/blob/f22d202cde2018f9455dec755118a9b84586e082/Vault_PrimaryLogo_Black.png">
```

Reply:
Just make sure it points to [`logo-hashicorp.svg`](https://github.com/hashicorp/nomad/blob/master/website/public/img/logo-hashicorp.svg) so it has **HashiCorp** in it.